src/flake8_bugbear/plugins/mutable_argument_default.rs

```diff
@@ -0,0 +1,62 @@
+use rustpython_ast::{Arguments, ExprKind};
+
+use crate::ast::types::{CheckLocator, Range};
+use crate::check_ast::Checker;
+use crate::checks::{Check, CheckKind};
+
+/// B006
+pub fn mutable_argument_default(checker: &mut Checker, arguments: &Arguments) {
+    for default in arguments
```

Comment:
Nit: can we rename `default` to `arg` or `expr`? `default` is a keyword so the syntax highlighting is confusing here.

Reply:
Got it, will change this.

Reply:
How about `deafult_value`?

Reply:
Chose `expr`.